spec/rails_helper.rb

```diff
@@ -50,6 +50,8 @@
   config.include TransactionHelpers
   config.include ExportHelpers
   config.include Devise::Test::ControllerHelpers, type: :controller
+  config.include FactoryHelpers
```

Comment:
The only place we need the `extend` (and to make this a class method) is in the `LinkedActivityValidator` spec. We're testing every level of activity in that file, which I don't think is necessary as there's no level-based validation (other than fund). We can simplify those tests and only test one level, rather than looping through them, which will mean we can sidestep this issue and remove the class method here.